testsuite/integration/web/pom.xml

```diff
@@ -687,5 +687,487 @@
+                                            <groupId>${testsuite.ee.galleon.pack.groupId}</groupId>
+                                            <artifactId>${testsuite.ee.galleon.pack.artifactId}</artifactId>
+                                            <version>${testsuite.ee.galleon.pack.version}</version>
+                                            <inherit-configs>false</inherit-configs>
+                                            <inherit-packages>false</inherit-packages>
+                                        </feature-pack>
+                                    </feature-packs>
+                                    <layers>
+                                        <layer>cloud-server</layer>
+                                    </layers>
+                                    <extra-server-content-dirs>
+                                        <extra-content>${basedir}/target/wildfly-users</extra-content>
+                                        <extra-content>${basedir}/target/external-taglib-module</extra-content>
+                                    </extra-server-content-dirs>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+
+
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <executions>
+                            <!-- Disable default-test -->
+                            <execution>
+                                <id>default-test</id>
+                                <phase>none</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                            </execution>
+
+                            <!-- Tests against the datasource-web-server without legacy security -->
+                            <execution>
+                                <id>bootable-jar-web-profile-layers.surefire</id>
+                                <phase>test</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration>
+                                    <environmentVariables>
+                                        <JBOSS_HOME>${jboss.dist}</JBOSS_HOME>
+                                    </environmentVariables>
+                                    <systemPropertyVariables>
+                                        <bootable.jar>${project.build.directory}/test-wildfly-web-profile.jar</bootable.jar>
+                                        <arquillian.xml>arquillian-bootable.xml</arquillian.xml>
+                                    </systemPropertyVariables>
+                                    <redirectTestOutputToFile>true</redirectTestOutputToFile>
```

Comment:
I've seen this around quite often in the TS as well but given what we can read in the community doc [1] I think this should be set to `${testLogToFile}` at least in new implementations, WDYT?

[1]
https://docs.wildfly.org/20/Testsuite.html#output-to-console

Reply:
Makes sense, thanks! I just copied it from Galleon profile, though, as you say, this is a new code.